index.js

```diff
@@ -4,23 +4,29 @@ var osTmpdir = require('os-tmpdir');
 var fs = require('graceful-fs');
 var mkdirp = require('mkdirp');
 var uuid = require('uuid');
+var PinkiePromise = require('pinkie-promise');
```

Comment:
Wasn't sure about naming a variable the same as a global variable (if Promise exists in for instance Node 4.x). But no problem :).